# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/WebRenderAPI.h
# Commit: f215196f1d1c
# Full Hash: f215196f1d1c8c6b42214b35fee351ef021bd704
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2020-03-28 09:32:56
# Description:
#   Bug 1624125 - Track display list changes in DisplayItemCache r=jrmuizel
#   
#   This is needed because display lists and DisplayItemCache have different lifetimes. For example, display lists can outlive WebRenderLayerManager when device reset occurs.
#   
#   A slightly nicer way of fixing this would be to couple DisplayItemCache with nsDisplayList or nsDisplayListBuilder. This is would currently require a lot of refactoring to look nice, because the painting code still supports non-retained display lists and non-WR code paths.
# ==============================================================================

diff -r 13a7265ed731 -r f215196f1d1c gfx/webrender_bindings/WebRenderAPI.h
--- a/gfx/webrender_bindings/WebRenderAPI.h	Fri Mar 27 16:49:40 2020 +0000
+++ b/gfx/webrender_bindings/WebRenderAPI.h	Fri Mar 27 16:49:37 2020 +0000
@@ -641,9 +641,6 @@
    */
   bool ReuseItem(nsPaintedDisplayItem* aItem);
 
-  void UpdateCacheState(const bool aPartialDisplayListBuildFailed);
-  void UpdateCacheSize();
-
   uint64_t CurrentClipChainId() const {
     return mCurrentSpaceAndClipChain.clip_chain;
   }
